### PR TITLE
Further deprecations to options in CREATE DATABASE

### PR DESCRIPTION
**Pull request by mnd999**
* Deprecate seedCredentials and seedConfig options without a (language) replacement.
* Deprecate standard and high_limit store formats

- [x] Still waiting on information from cluster as to how to do seeding from buckets now. 

Depends on: https://github.com/neo4j/docs-operations/pull/1909

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1083.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1083.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 1]
```diff
@@ -16,6 +16,38 @@ New features are added to the language continuously, and occasionally, some feat
...
+| Details
+
+a|
+label:functionality[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CREATE DATABASE db OPTIONS { seedCredentials: ...,  seedConfig: ... }
+----
+| The `CREATE DATABASE` options `seedCredentials` and `seedConfig` have been deprecated. For seeding from cloud storage, it is recommended to use `CloudSeedProvider` which will read cloud credentials and configuration from standard locations. For further information see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/clustering/databases/#_seed_providers[Seed providers]
```
Comment (Hunterness) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
| The `CREATE DATABASE` options `seedCredentials` and `seedConfig` have been deprecated.
For seeding from cloud storage, it is recommended to use `CloudSeedProvider` which will read cloud credentials and configuration from standard locations.
For further information see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/clustering/databases/#_seed_providers[Seed providers].
```
let's split the sentences over different lines and add a dot at the end of the last one

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 2]
```diff
@@ -16,6 +16,38 @@ New features are added to the language continuously, and occasionally, some feat
...
+label:functionality[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CREATE DATABASE db OPTIONS { storeFormat: 'standard' }
+
+CREATE DATABASE db OPTIONS { storeFormat: 'high_limit' }
+----
+| The standard and high_limit store formats have been deprecated. Creating databases with these formats is therefore also deprecated.
+For more information on the deprecation of these formats, see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-internals/store-formats/#format-deprecations[Format deprecations]
```
Comment (Hunterness) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
For more information on the deprecation of these formats, see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-internals/store-formats/#format-deprecations[Format deprecations].
```

**Review by Hunterness — APPROVED** [review]
Just two missed `.`

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1083.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1083.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 3]
```diff
@@ -16,6 +16,40 @@ New features are added to the language continuously, and occasionally, some feat
...
+a|
+label:functionality[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CREATE DATABASE db OPTIONS { seedCredentials: ...,  seedConfig: ... }
+----
+| The `CREATE DATABASE` options `seedCredentials` and `seedConfig` have been deprecated.
+For seeding from cloud storage, it is recommended to use `CloudSeedProvider` which will read cloud credentials and configuration from standard locations.
+For further information see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/clustering/databases/#_seed_providers[Seed providers].
```
Comment (jackwaudby) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: This link will need to be updated to `#_cloud_seed_provider[CloudSeedProvider]`

**Review by jackwaudby — APPROVED** [review]

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc (hunk 3 above)
Comment (mnd999) [reply modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: Updated, shall I do this in the operations manual as well?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1083.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1083.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1083.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1083.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1083.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1083.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc (hunk 3 above)
Comment (mnd999) [reply modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: Turns out it's `#cloud-seed-provider`

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1083.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1083.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by Hunterness — APPROVED** [review]

**Comment by mnd999** [comment]
Merged here - https://github.com/neo-technology/neo4j/pull/28042

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1083.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1083.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**